deployment-examples/gcp/README.md

```diff
@@ -24,12 +22,12 @@ Google Cloud Project. It requires to have:
 |`GCLOUD_CLUSTER_NAME` | Existing Google Kubernetes Engine cluster name | YES | `adyen-demo` |
```

Comment:
cluster name should not have a default value. can you check both table and readme table and make them consistent to each other.

Reply:
34666127b77bfce2ee7c41aae6209b6ae1020131

Reply:
Having hard to time to diff when multiple comments addressed in one commit, I will review from stratch.